<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_dialog</class>
 <widget class="QDialog" name="search_dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>6</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>鹅山通</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.svg</normaloff>icon.svg</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	color: black;
	border: 2px solid #33373E; /* 边框宽度和颜色 */
	border-radius: 10px; /* 边框圆度 */;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,2,1">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="618,100">
        <item>
         <widget class="QLineEdit" name="startText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid #33373E;      
    border-radius: 10px;            
    padding-left: 10px;             
    padding-right: 10px;            
    color: #000000;                 
    background-color: #FFFFFF;      
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>请输入起点站的完整名称</string>
          </property>
          <property name="cursorMoveStyle">
           <enum>Qt::LogicalMoveStyle</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchStart">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 10px;
	border: 2px solid #33373E;
    background-color: #FFFFFF;      
}
QPushButton:hover {
    background-color: rgb(209, 209, 209); 
}
QPushButton:pressed {
    background-color: rgb(190, 190, 190); 
}</string>
          </property>
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="618,100">
        <item>
         <widget class="QLineEdit" name="endText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid #33373E;      
    border-radius: 10px;            
    padding-left: 10px;             
    padding-right: 10px;            
    color: #000000;                 
    background-color: #FFFFFF;      
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>请输入终点站的完整名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchEnd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 10px;
	border: 2px solid #33373E;
    background-color: #FFFFFF;      
}
QPushButton:hover {
    background-color: rgb(209, 209, 209); 
}
QPushButton:pressed {
    background-color: rgb(190, 190, 190); 
}</string>
          </property>
          <property name="text">
           <string>选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="mapOpen">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 10px;
	border: 2px solid #33373E;
    background-color: white;
}
QPushButton:hover {
    background-color: rgb(209, 209, 209); 
}
QPushButton:pressed {
    background-color: rgb(190, 190, 190); 
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>map_button.png</normaloff>map_button.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>500</width>
          <height>115</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="search">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 10px;
	border: 2px solid #33373E;
    background-color: #FFFFFF;      
}
QPushButton:hover {
    background-color: rgb(209, 209, 209); 
}
QPushButton:pressed {
    background-color: rgb(190, 190, 190); 
}</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border-radius: 10px;
	border: 2px solid #33373E;
    background-color: #FFFFFF;      
}
QPushButton:hover {
    background-color: rgb(209, 209, 209); 
}
QPushButton:pressed {
    background-color: rgb(190, 190, 190); 
}</string>
          </property>
          <property name="text">
           <string>选择模式</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
